OracleOpenJDK/14/Dockerfile

```diff
@@ -10,76 +10,63 @@
 # ----------------------------------
 #
 # (1) openjdk-14.xx_linux-x64_bin.tar.gz
-#     Download from https://jdk.java.net/14/
+#     Downloaded from https://jdk.java.net/14/
 #
 # HOW TO BUILD THIS IMAGE
 # -----------------------
-# Put all downloaded files in the same directory as this Dockerfile
 # Run:
 #      $ docker build -t oracle/openjdk:14 .
 #
 # This command is already scripted on build.sh so you can alternatively run
 #		$ bash build.sh
 #
-# The builder image will be use to uncompress the tar.gz file with the Java Runtime.
 
-FROM oraclelinux:7-slim as builder
+FROM oraclelinux:7-slim
 
 MAINTAINER Aurelio Garcia-Ribeyro <aurelio.garciaribeyro@oracle.com>
 
+
+ENV JAVA_PKG=https://download.java.net/java/GA/jdk14.0.2/205943a0976c4ed48cb16f1043c5c647/12/GPL/openjdk-14.0.2_linux-x64_bin.tar.gz \
+	JAVA_SHA256=91310200f072045dc6cef2c8c23e7e6387b37c46e9de49623ce0fa461a24623d \
+	JAVA_VERSION=14.0.2 \
+	JAVA_HOME=/usr/java/jdk-14 
+
+ENV	PATH $JAVA_HOME/bin:$PATH
+
+ENV LANG en_US.UTF-8
```

Comment:
This directive should be added to the one on line 19 to remove a layer from the final image.